### PR TITLE
Fix build m1

### PR DESCRIPTION
**Pull request by last-partizan**
Attempt to fix m1 build

## What kind of change does this PR introduce?
- [x] Fix
- [ ] Feature
- [ ] Codestyle
- [ ] Refactor
- [ ] Other

## Did this PR introduce a breaking change?
_A breaking change includes anything that breaks backwards compatibility either at compile or run time._
- [ ] Yes, please list breaking changes
- [x] No

### CODE REVIEW

File: .github/workflows/snap.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: Snap
 
-on: [push]
+on:
+  push:
+    branches: [ main ]
```
Comment (last-partizan) [inline .github/workflows/snap.yaml:5]: @Kethku i was fixing build for m1, when i noticed this. Bulding and publishing snaps on each push.

That's probably not how it supposed to work. I changed this to publish snaps only for main branch.

Comment (Kethku) [reply .github/workflows/snap.yaml:5]: I think you're likely correct. Good catch

**Comment by Kethku** [comment]
I have no reason to assume this doesn't fix things, but I really don't have the expertise to know either way. @last-partizan I'll defer to your judgement. THis looks good to me.

**Comment by last-partizan** [comment]
Great, merging thins.

And it will publish snaps after merge.

**Comment by j4qfrost** [comment]
This was my bad, the on push was there because I was testing my changes.

**Comment by last-partizan** [comment]
Btw, pipeline completed successfully, but latest release here: https://snapcraft.io/neovide is still from `13 June 2021`.

Any ideas why?

**Comment by shaunsingh** [comment]
I believe that release is still at the stable version, there should be a flag to install the latest version published by the actions 

I believe it was `--beta`

**Comment by j4qfrost** [comment]
I have to manually add the snaps in. Not sure there's a way around that.
Stable releases should be added manually anyway.

**Pull request merged**